assets/js/theme/search.js

```diff
@@ -132,8 +132,43 @@ export default class Search extends CatalogPage {
         $($tabsCollection.get(nextTabIdx)).focus().trigger('click');
     }
 
+    getUrlParameter(queryParam) {
+        const queryParamKey = queryParam.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
+        const regex = new RegExp(`[\\?&]${queryParamKey}=([^&#]*)`);
+        const results = regex.exec(window.location.search);
+        return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
+    }
+
+    setupSortByQuerySearchParam() {
+        const searchQuery = this.getUrlParameter('search_query');
+
+        if (searchQuery.length === 0) return;
+
+        const $baseInput = $('<input/>')
+            .attr({
+                type: 'text',
+                class: 'aria-description--hidden',
```

Comment:
I believe it's not semantic to use aria-description--hidden class here

Reply:
Maybe we can try use input with type hidden here instead